scripts/skills/perks/perk_rf_second_wind.nut

```diff
@@ -0,0 +1,31 @@
+this.perk_rf_second_wind <- ::inherit("scripts/skills/skill", {
+	m = {
+		RecoverActionPointTarget = 4
```

Comment:
`ActionPoints` is always plural so I'd rewrite this as `RecoverActionPointsTarget` and in fact you could make it even shorter and just write `ActionPointsTarget` and that's enough because we know from the perk itself that it's gonna recover it to that target.